src/Types/ByteArray.js

```diff
@@ -0,0 +1,64 @@
+// From instance for Array in prelude
+exports.ordImpl = function (f) {
```

Comment:
Minor nitpick, the rest of the codebase prepends FFIs with an underscore, we should probably stick to one convention.